src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -2,15 +2,11 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-import java.util.Properties;
+import java.util.*;
```

Comment:
I understand it might be added automatically from your IDE. Could you please revert these wildcard import changes? It becomes difficult to manage conflicts with these.